examples/disable-stdout-logging/main.tf

```diff
@@ -10,6 +10,6 @@ resource "lacework_agent_access_token" "k8s" {
 module "lacework_k8s_datacollector" {
   source = "../../"
 
-  lacework_access_token    = lacework_agent_access_token.k8s.token
+  lacework_access_token     = lacework_agent_access_token.k8s.token
```

Comment:
Could you remove this change from the PR?

Reply:
https://github.com/lacework/terraform-kubernetes-agent/pull/72

Merge that one, then I'll rebase or something.